src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/updatecontactdetails/UpdateContactDetailsAboutToStartHandlerTest.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.handler.updatecontactdetails;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.provider.Arguments;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.finrem.caseorchestration.FinremCallbackRequestFactory;
+import uk.gov.hmcts.reform.finrem.caseorchestration.ccd.callback.CallbackType;
+import uk.gov.hmcts.reform.finrem.caseorchestration.handler.FinremCallbackRequest;
+import uk.gov.hmcts.reform.finrem.caseorchestration.mapper.FinremCaseDetailsMapper;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.EventType;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CaseType;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.FinremCaseData;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.NoticeOfChangeParty;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.YesOrNo;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.wrapper.ContactDetailsWrapper;
+
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.TestConstants.AUTH_TOKEN;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.test.Assertions.assertCanHandle;
+
+@ExtendWith(MockitoExtension.class)
+class UpdateContactDetailsAboutToStartHandlerTest {
+
+    @InjectMocks
+    private UpdateContactDetailsAboutToStartHandler handler;
+
+    @Mock
+    private FinremCaseDetailsMapper finremCaseDetailsMapper;
+
+    @Test
+    void canHandle() {
+        assertCanHandle(handler,
+            Arguments.of(CallbackType.ABOUT_TO_START, CaseType.CONSENTED, EventType.UPDATE_CONTACT_DETAILS),
+            Arguments.of(CallbackType.ABOUT_TO_START, CaseType.CONTESTED, EventType.UPDATE_CONTACT_DETAILS)
+        );
+    }
+
+    @Test
+    void testHandle() {
+        FinremCaseData caseData = FinremCaseData.builder().contactDetailsWrapper(
+            ContactDetailsWrapper.builder()
+                .nocParty(NoticeOfChangeParty.APPLICANT)
+                .updateIncludesRepresentativeChange(YesOrNo.YES)
+                .build()).build();
+
+        FinremCallbackRequest request = FinremCallbackRequestFactory.from(caseData);
+
+        var response = handler.handle(request, AUTH_TOKEN);
+        assertNull(response.getData().getContactDetailsWrapper().getNocParty());
+        assertNull(response.getData().getContactDetailsWrapper().getNocParty());
+    }
```

Comment:
Asserting the same thing twice.  

Second assertion should be for updateIncludesRepresentativeChange I reckon.